cmd/fluent-bit/config.go

```diff
@@ -258,5 +259,31 @@ func parseConfig(cfg ConfigGetter) (*config, error) {
 		res.bufferConfig.dqueConfig.queueName = queueName
 	}
 
+	res.clientConfig.Client.TLSConfig.CAFile = cfg.Get("ca_file")
```

Comment:
Do you need a rebase or you are also introducing this change ? I would rather have a separate PR for a better changelog.

Reply:
yes, I create another PR for mTLS https://github.com/grafana/loki/pull/2568

Reply:
Awesome let's get mTLS merge first and then rebase.